pypeit/core/pydl.py

```diff
@@ -915,7 +915,7 @@ def djs_reject(data, model, outmask=None, inmask=None,
     # to python True and False booleans
 
     outmask = newmask
-    return (outmask, qdone)
+    return outmask, qdone
```

Comment:
I think you are correct, but outmask is also an  input argument that is mutable. That is generally a poor design, but I'm loathe to mess with this now since it could break all our rejection.